dnf5-plugins/builddep_plugin/builddep.cpp

```diff
@@ -272,7 +282,8 @@ void BuildDepCommand::run() {
             // we do not download filelists and some files could be explicitly mentioned in provide section. The best
             // solution would be to merge result of provide and file search to prevent problems caused by modification
             // during distro lifecycle.
-            goal->add_rpm_install(spec, settings);
+            const auto & escaped_spec = escape_glob(spec);
```

Comment:
Would you mind adding a TODO to remove the escaping once we have a setting to switch off the globs expanding? Maybe also add a link to the issue.

Reply:
Done!